packages/sanity/src/core/hooks/useDocumentOperation.ts

```diff
@@ -1,12 +1,18 @@
-import {useMemoObservable} from 'react-rx'
+import {useMemo} from 'react'
+import {useObservable} from 'react-rx'
 
 import {type OperationsAPI, useDocumentStore} from '../store'
 
 /** @internal */
 export function useDocumentOperation(publishedDocId: string, docTypeName: string): OperationsAPI {
   const documentStore = useDocumentStore()
-  return useMemoObservable(
+  const observable = useMemo(
     () => documentStore.pair.editOperations(publishedDocId, docTypeName),
     [docTypeName, documentStore.pair, publishedDocId],
-  )!
+  )
+  /**
+   * We know that since the observable has a startWith operator, it will always emit a value
+   * and that's why the non-null assertion is used here
+   */
+  return useObservable(observable)!
```

Comment:
I tried to set an `initialValue` here instead, but couldn't work out what that initial value looks like 🤔

Reply:
It's fine - thanks for adding the comment though!